slam-ws/src/nav_goal/bin/movebase_client_map

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env python
+
+import nav_goal.movebase_client
+import rospy
+import sys
+
+if __name__ == '__main__':
+    rospy.init_node('movebase_client_py')
+
+    if len(sys.argv) < 4:
+        rospy.logerr("Error: Insufficient Arguments - Usage is 'rosrun nav_goal movebase_client x y yaw' where x is distance to move in x, y is distance to move in y, and yaw is rotation in radians.")
+        rospy.signal_shutdown("Error: Insufficient Arguments - Usage is 'rosrun nav_goal movebase_client x y yaw' where x is distance to move in x, y is distance to move in y, and yaw is rotation in radians.")
+    else:
+        try:
+            x = float(sys.argv[1])
+            y = float(sys.argv[2])
+            yaw = float(sys.argv[3])
+        except:
+            rospy.logerr("Error: Incorrect Argument Types - All 3 arguments must be floats or integers.")
+            rospy.signal_shutdown("Error: Incorrect Argument Types - All 3 arguments must be floats or integers.")
+            sys.exit(1)
+
+       	result = nav_goal.movebase_client.movebase_client(x, y, yaw, "map")
+       	rospy.loginfo(result)
```

Comment:
In general, this code is a lot of copy paste from`bin/movebase_client`. Can we just call `bin/movebase_client` from `bin/movebase_client_map` and just specify input parameters? If you can't call it then I would suggest turning the whole argument parsing and calling `movebase_client()` into a function in `movebase_client.py` and just call this function in each of the executable's `__main__`

Reply:
Done